src/extensions/default/bramble/lib/ConsoleManagerRemote.js

```diff
@@ -0,0 +1,64 @@
+(function(transport) {
+    "use strict";
+
+    //TODO: add support for console.log(one,two, three) arguments
+
+    // Returns a function which calls the specific function in scope
+    var bind = function(func, scope, args) {
```

Comment:
You are absolutely right, it was not being used anywhere. It was written while I was migrating a Console shim suggested, but I forgot entirely to delete it since the shim did not need the aforementioned ```bind()``` function to properly work. Removed in upcoming commit.